ES/ESum042.xml

```diff
@@ -644,250 +355,425 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                            <title type="complete" ref="LIT4669TaammeraQi">Miracle 2. About a believer in a church dedicated to Qirqos</title>
+                            <textLang mainLang="gez"/>                            
+                         </msItem>
+                         
+                         <msItem xml:id="p3_i1.3">
+                            <locus from="140vb" to="141rb"/>
+                            <title type="complete" ref="LIT4669TaammeraQi">Miracle 3. About a poor monk from a distant country</title>
+                            <textLang mainLang="gez"/>                            
+                         </msItem>
+                                                  
+                         <msItem xml:id="p3_i1.4">
+                            <locus from="141rb" to="141vb"/>
+                            <title type="complete" ref="LIT4669TaammeraQi">Miracle 4. About a monk called Eli (ʾElyās)</title>
+                            <textLang mainLang="gez"/> 
+                            <note>This Miracle is added by <ref target="#h2">Hand 2</ref> (of <persName role= "scribe" ref="PRS13967WaldaTensae">Walda Tǝnśāʾe</persName>).</note>
+                         </msItem>                        
+                         </msItem>                                           
+                      
+                      <msItem xml:id="p3_i2">
+                         <locus target="#143ra"/>
+                         <title type="incomplete" ref="LIT4714Salamlaki">Salutation to you, treshold of the East...</title>
+                         <textLang mainLang="gez"/>
+                         <note>In <ref target="#h2">Hand 2</ref> <persName ref="PRS13967WaldaTensae">Walda Tǝnśāʾe</persName>. There are two words in the upper line of the 
+                            folium looking like the end of another text that could not be identified (possible there was another leaf befor which is missing now).
+                             </note>
+                      </msItem>
+                      
+                      <msItem xml:id="p3_i3">
+                         <locus from="143ra" to="144ra"/>
+                         <title type="complete" ref="LIT4717TaammerLi">One miracle of Libānos</title>
+                         <textLang mainLang="gez"/>
+                         <incipit xml:lang="gez">ተአምሪሁ፡ ለአቡነ፡ ሊባኖስ፡ ጸሎቱ፡ ወበረከቱ፡ የሀሉ፡ መስለ፡ ገብ<cb n="b"/>ሩ፡ 
+                            ክፍለ፡ ማርያም፡ ለዓለመ፡ ዓለም፡ አሜን። ወካዕበ፡ ይቢሉኒ፡ እንዘ፡ ሀሎ፡ ስርናይ፡ ምልዕልተ፡ ናሕስ፡ ዘቤተ፤ ክርስቲያንኑ፤ ዘአንበርዎ፡ ወድዮሙ፡ ሐሰል፡ ከመ፡ ይትዓቀብ፡ ህየ፡</incipit>                         
+                      </msItem>
+                      
+                      <msItem xml:id="p3_i4">
+                         <locus from="144ra" to="144vb"/>
+                         <locus from="142ra" to="142rb"/>
+                         <title type="complete" ref="LIT3941Miracle">Miracle of Jesus: Healing of the blind man by Jesus</title>
```

Comment:
I would wrap this inside of an msItem pointing to LIT2382Taamme so that we find it when we look for the MJ mss

Reply:
ok